### ISSUE TITLE
configure.ac: checks missing for 'DPDK > 16.04' when building dpdkstats

### ISSUE THREAD

**Issue by rjablonx** [collectd/collectd#2989]
*   Version of collectd: 5.8 (also applicable on 5.8.1 and master)
*   Operating system / distribution: ubuntu 16.04
*   Kernel version (if applicable):

## Expected behavior

dpdkstat plugin should not be build if DPDK on the host system is older than 16.04 

## Actual behavior

dpdkstat plugin is marked as one of the features to be build even if DPDK on the host system is older than 16.04 (it is hard to check what version is exactly - it is one of CI nodes for OPNFV Barometer project but since te_eal_primary_proc_alive is missing it is probably older than DPDK16.04)
In the configure logs we can see that dpdkstats build fails:

>  CC       src/dpdkstat_la-utils_dpdk.lo
> src/utils_dpdk.c: In function ‘dpdk_helper_cmd_wait’:
> src/utils_dpdk.c:585:8: error: implicit declaration of function ‘rte_eal_primary_proc_alive’ [-Werror=implicit-function-declaration]
>    if (!rte_eal_primary_proc_alive(phc->eal_config.file_prefix)) {
>         ^
> cc1: all warnings being treated as errors
> Makefile:5939: recipe for target 'src/dpdkstat_la-utils_dpdk.lo' failed

Earlier we can see that dpdkevents plugin was omitted during configure phase but dpdkstats was selected for build:
> dpdkevents. . . . . . no (DPDK version < 16.07)
> dpdkstat  . . . . . . yes

rte_eal_primary_proc_alive() was introduced in DPDK 16.04 so I think that configure.ac should check for it before building dpdkstats (just like dpdkevents checks for DPDK 16.07 using $dpdk_keepalive variable)

**Comment by atyronesmith** [comment]
dpdkstats is targeted for the build because it lacks the necessary checks to ensure a proper DPDK version.

**Comment by rubenk** [comment]
We do document that you need at least DPDK 16.04 in https://github.com/collectd/collectd/blob/master/docs/BUILD.dpdkstat.md

**Comment by atyronesmith** [comment]
Yes.  The collectd documentation is correct.  I was referring to the actual code for dpdkstats needing a check to mirror what is done in dpdkevents.  This issue is related to build issues we are having in the OPNFV Barometer project and is related to an out of date DPDK ending up on the build server.

**Comment by rubenk** [comment]
It would indeed be nicer to fail at configure time instead of compile time.
I'm willing to accept a PR that fixes that. Probably easiest is to just check for libdpdk >= 16.07 unconditionally for both dpdkevents and dpdkstat and bump the minimum required version.
